### PR TITLE
rm: factory: Add ci-scripts notes

### PR DESCRIPTION
**Pull request by angolini**
Signed-off-by: Daiane Angolini <daiane.angolini@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 93215b953dcabaa8496b5a14327f33ec8b7adb30 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1064/docs/artifacts/html/index.html

File: source/reference-manual/factory/factory-sources.rst [hunk 1]
```diff
@@ -40,6 +40,10 @@ You will find four git repositories, below is a brief description of each one.
    ci-scripts.git
      Defines your platform and container build job to our continuous integration system.
 
+     .. note:: 
+        * The **ci-scripts.git** can only have ``master`` branch.
+        * The git history from **ci-scripts.git** cannot be override (force push is disabled).
```
Comment (vanmaegima) [inline source/reference-manual/factory/factory-sources.rst:45]:
```suggestion
        * The git history from **ci-scripts.git** cannot be overriden (force push is disabled).
```

**Review by vanmaegima — CHANGES_REQUESTED** [review]

File: source/reference-manual/factory/factory-sources.rst (hunk 1 above)
Comment (MrCry0) [reply source/reference-manual/factory/factory-sources.rst:45]: > 

@vanmaegima Maybe `overridden`?

**Comment by doanac** [comment]
Docs for 41ea5b96fa4469053e1f9bd594399c1cfbfa0f26 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1065/docs/artifacts/html/index.html

**Review by Tim-Anderson — APPROVED** [review]
LGTM

File: source/reference-manual/factory/factory-sources.rst (hunk 1 above)
Comment (angolini) [reply source/reference-manual/factory/factory-sources.rst]: the new version does not have this word so I'm solving this

**Review by vanmaegima — APPROVED** [review]
LGTM

**Review by munoz0raul — APPROVED** [review]
LGTM

**Pull request merged**